connector/connect/server/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -1061,6 +1068,81 @@ class SparkConnectPlanner(session: SparkSession) {
     }
   }
 
+  private def transformNumericAggregate(agg: proto.Aggregate): LogicalPlan = {
+    if (!agg.hasInput) {
+      throw InvalidPlanInput("Aggregate needs a plan input")
+    }
+    if (agg.getAggregateExpressionsCount == 0) {
```

Comment:
wouldn't this be caught by the analyzer?

Reply:
this protocol `functionName: String, aggCols: Seq[String]` is just defined in this PR, we have to process it by ourself